shared/actions/kbfs/index.platform.desktop.js

```diff
@@ -315,17 +317,28 @@ function* openInWindows(openPath: string): SagaGenerator<any, any> {
 
 function* openSaga(action: FSOpen): SagaGenerator<any, any> {
   const openPath = action.payload.path || Constants.defaultKBFSPath
-
-  console.log('openInKBFS:', openPath)
-  if (isWindows) {
-    yield* openInWindows(openPath)
+  const enabled = yield select(state => state.favorite.fuseStatus && state.favorite.fuseStatus.kextStarted)
+  if (enabled) {
+    console.log('openInKBFS:', openPath)
+    if (isWindows) {
+      yield* openInWindows(openPath)
+    } else {
+      yield call(openInDefault, openPath)
+    }
   } else {
-    yield call(openInDefault, openPath)
+    yield put(navigateTo([], [folderTab]))
+    yield put(switchTo([folderTab]))
```

Comment:
we don't need to make this change but i believe you can switch and route just using switchTo